origin-bridge/logic/attestation_service.py

```diff
@@ -398,16 +399,15 @@ def verify_twitter(oauth_verifier, eth_address):
                 'The verifier you provided is invalid.'
             )
 
-        # TODO: determine what the text should be
-        data = 'twitter verified'
-        # TODO: determine claim type integer code for phone verification
-        signature = attestations.generate_signature(
-            signing_key, eth_address, CLAIM_TYPES['twitter'], data
-        )
-
         query_string = urllib.parse.parse_qs(response.content)
         screen_name = query_string[b'screen_name'][0].decode('utf-8')
 
+        ipfs_hash = ipfs_helper.pin_json({'screen_name': screen_name})
```

Comment:
To be consistent with all the other data we are storing in IPFS that uses JSON schema and also to allow future validation and versioning, let's store a schemaID. I'd suggest something like this:
```
{
  "schemaId": "http://schema.originprotocol.com/twitter-attestation_v1.0.0",
  "screen_name": screen_name
}
```

Reply:
good point 👍